### PR TITLE
Add license, run details and update quarkus

### PR DESCRIPTION
**Pull request by PerfectSlayer**
I confirm you can pick another winner without query twitter is working well.

### CODE REVIEW

**Comment by sunix** [comment]
Are you sure you want to use GPL ?

**Comment by sunix** [comment]
are you sure we want to use GPL?

File: README.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+mvn package
 ```
 
 ## Run
 
+Run providing Twitter client credentials:
+
 ```bash
-./mvn compile quarkus:dev
+java -Dtwitter4j.oauth.consumerKey=<consumerKey> -Dtwitter4j.oauth.consumerSecret=<consumerSecret> -Dtwitter4j.oauth.accessToken=<accessToken> -Dtwitter4j.oauth.accessTokenSecret=<accessTokenSecret> -jar target/twitter-raffle-1.0.0-SNAPSHOT-runner.jar
```
Comment (sunix) [inline README.md:16]: Could you also add how to run with env variable? 
http://twitter4j.org/en/configuration.html#environmentvariableconfiguration

**Review by sunix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sunix) [reply README.md:16]: forget about this, it doesn't work: https://github.com/Twitter4J/Twitter4J/issues/361

**Review by sunix — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (PerfectSlayer) [reply README.md:16]: Too late, I already add it

**Comment by PerfectSlayer** [comment]
Yeah, I'm sure about GPL.

**Pull request merged**